gcloud-java-resourcemanager/src/main/java/com/google/gcloud/spi/ResourceManagerRpc.java

```diff
@@ -29,13 +29,24 @@
 public interface ResourceManagerRpc {
 
   public enum Permission {
-    CREATE,
-    DELETE,
-    GET,
-    LIST,
-    REPLACE,
-    GET_IAM_POLICY,
-    REPLACE_IAM_POLICY
+    CREATE("resourcemanager.projects.create"),
```

Comment:
The more I think about it the more it makes sense to me (and I think we should discuss it) that
Policy management should be decoupled from this service (and live in it's own module - even
if it happen to use the same underlying service).

Even the public docs suggests that "resource-manager" is *one* of the Cloud Platform products
that currently support IAM.

Permission list, and role set seems to be service specific (even if they have some overlaps).